gradle/libs.versions.toml

```diff
@@ -74,6 +74,7 @@ smallrye-common-annotation = { module = "io.smallrye.common:smallrye-common-anno
 spotbugs-annotations = { module = "com.github.spotbugs:spotbugs-annotations", version = "4.8.5" }
 swagger-annotations = { module = "io.swagger:swagger-annotations", version.ref = "swagger" }
 swagger-jaxrs = { module = "io.swagger:swagger-jaxrs", version.ref = "swagger" }
+jaxrs-api = { module = "jakarta.ws.rs:jakarta.ws.rs-api", version = "3.0.0" }
```

Comment:
This lib is already declared as `jakarta-ws-rs-api`.

Reply:
thx! somehow I missed that :facepalm:

Reply:
fixed